src/System.ServiceModel.Tests.Common/src/OuterLoopTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using Newtonsoft.Json;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+
+namespace System.ServiceModel.Tests.Common
+{
+    public abstract class OuterLoopTests : IDisposable
+    {
+        protected string EndpointAddress
+        {
+            get;
+            private set;
+        }
+
+        private string BaseAddress
+        {
+            // Would like to pull this address either from msbuild properties, env vars, or 
+            // configuration passed into xunit.
```

Comment:
Are you talking about the BaseAddress? That's the address to the bridge. The EndpointAddress is coming from the bridge.

-----Original Message-----
From: "Sajay Antony" <notifications@github.com>
Sent: ‎6/‎24/‎2015 10:13 AM
To: "dotnet/wcf" <wcf@noreply.github.com>
Cc: "Dustin Metzgar" <dustin.metzgar@gmail.com>
Subject: Re: [wcf] Bridge test changes (#115)

In src/System.ServiceModel.Tests.Common/src/OuterLoopTests.cs:
> +using System.Text;
> +
> +namespace System.ServiceModel.Tests.Common
> +{
> +    public abstract class OuterLoopTests : IDisposable
> +    {
> +        protected string EndpointAddress
> +        {
> +            get;
> +            private set;
> +        }
> +
> +        private string BaseAddress
> +        {
> +            // Would like to pull this address either from msbuild properties, env vars, or 
> +            // configuration passed into xunit.
This should come from the bridge. 
—
Reply to this email directly or view it on GitHub.